force-app/main/default/lwc/paycheckCalculator/paycheckCalculator.js

```diff
@@ -0,0 +1,119 @@
+import { LightningElement, track } from 'lwc';
+
+export default class PaycheckCalculator extends LightningElement {
+
+    annualSalary = 0;
+    monthlyGrossSalary = 0;
+    biWeeklyGrossSalary = 0;
+    monthlyNetSalary = 0;
+    biWeeklyNetSalary = 0;
+
+    @track federalIncomeTax = 0;
+    @track socialSecurityTax = 0;
+    @track medicareWithholding = 0;
+    @track takeHomePay = 0;
+
+    federalTaxBrackets = [
+        { limit: 11000, rate: 0.1 },
+        { limit: 44725, rate: 0.12 },
+        { limit: 95375, rate: 0.22 },
+        { limit: 182100, rate: 0.24 },
+        { limit: 231250, rate: 0.32 },
+        { limit: 578125, rate: 0.35 },
+        { limit: Infinity, rate: 0.37 }
+    ];
+
+    // Columns for Salary Breakdown
+    salaryColumns = [
+        { label: 'Salary Type', fieldName: 'label' },
+        { label: 'Amount', fieldName: 'value', type: 'currency' }
+    ];
+
+    // Columns for Tax Information
+    taxColumns = [
+        { label: 'Taxes Taken Out', fieldName: 'label' },
+        { label: 'Amount', fieldName: 'value', type: 'currency' },
+        {
+            label: 'Info',
+            type: 'button-icon',
+            fieldName: 'info',  // Info field to trigger the tooltip
+            typeAttributes: {
+                iconName: 'utility:info',
+                alternativeText: { fieldName: 'alternativeText' }, // hover text
+                variant: 'border-filled',
+                size: 'xx-small',
+                iconPosition: 'right'
+            }
+        }
+    ];
+
+    handleChange(event) {
+        this.annualSalary = Number(event.target.value);
```

Comment:
Maybe add some validation here (i.e. make sure annualSalary is always a positive number)